apps/Gruntfile.js

```diff
@@ -345,12 +345,18 @@ module.exports = function (grunt) {
           ' -p [ factor-bundle -o ' + allFilesDest.join(' -o ') + ' ] -o ' + outputDir + 'common.js' :
           ' -o ' + allFilesDest[0]
       );
+  var applabAPIExec = 'mkdir -p build/browserified && `npm bin`/browserifyinc' +
+      ' --cachefile ' + outputDir + 'applab-api-cache.json' +
+      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      (envOptions.dev ? '' : ' -t [ envify --NODE_ENV production ]') +
+      ' -d build/js/applab/api-entry.js -o ' + outputDir + 'applab-api.js';
```

Comment:
I could do that. It seems like it would be easier to just use the javascript api for browserify to do this instead of shelling out to the command line, but there is a comment above indicating that using the command line tools is faster and more stable. Don't the command line tools just use the javascript apis? why would the command line be faster and more stable?